tools/PI/DevHome.PI/Strings/en-us/Resources.resw

```diff
@@ -1069,6 +1065,14 @@
     <value>Project Ironsides needs to run as Admin in order to modify WER collection for an app.</value>
     <comment>{Locked="Ironsides"} Tooltip for a button that relaunches Project Ironsides as admin to change WER settings</comment>
   </data>
+  <data name="RunElevatedButton.Text" xml:space="preserve">
+    <value>Run as admin</value>

```

Comment:
The Icon already means "Run as admin". Want to go for something like "Get Information" to be a bit more descriptive?